packages/components/src/components/mixins/statusState.js

```diff
@@ -5,7 +5,7 @@ export default {
   props: {
     statusStates: {
       type: Array,
-      default: () => Array.from({ length: 5 }, () => 0),
+      default: () => Array.from({ length: 4 }, () => 0),
```

Comment:
For my edification, what are the `statusStates`?

Reply:
`statusStates` are how we keep track of the status of each step in the instructions flow. It's just an array of integers where `0` means not started (blank segment in the pie), `1` means started but not completed (yellow segment in the pie), and `2` means finished (green segment in the pie).